pkg/languagedetection/detector_nix_test.go

```diff
@@ -85,6 +85,31 @@ func TestLanguageFromCommandline(t *testing.T) {
 			cmdline:  []string{"dotnet", "myApp"},
 			expected: languagemodels.Dotnet,
 		},
+		{
+			name:     "rails",
+			cmdline:  []string{"bin/rails", "asdf"},
+			expected: languagemodels.Ruby,
+		},
+		{
+			name:     "ruby",
+			cmdline:  []string{"ruby", "myapp.rb"},
+			expected: languagemodels.Ruby,
+		},
+		{
+			name:     "rubyw",
+			cmdline:  []string{"rubyw", "myapp.rb"},
+			expected: languagemodels.Ruby,
+		},
+		{
+			name:     "ruby interactive shell",
+			cmdline:  []string{"irb"},
+			expected: languagemodels.Ruby,
+		},
+		{
+			name:     "sidekiq",
+			cmdline:  []string{"sidekiq"},
+			expected: languagemodels.Ruby,
+		},
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
We're testing only happy flows, but I do suggest to add negative flows, to ensure we don't have false positives, like `rubywow` shouldn't match, etc.

Reply:
Ah good suggestion, I will add it.